src/core/listener.c

```diff
@@ -748,6 +748,9 @@ QuicListenerAcceptConnection(
         return;
     }
 
+    Connection->FixedBit = 1;
+    Connection->IgnoreFixedBit = 0;
```

Comment:
Any non-zero defaults should be set in the connection initialization function. Better still, rework the variables so `0` is the default, so you don't have to do anything special.

Reply:
I assume the Connection Initialization function is `QuicConnAlloc`?

Reply:
Yes.